uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/implementation/AgentControlImpl.java

```diff
@@ -219,13 +219,30 @@ public MqttSubscribeReply unsubscribeMqtt(MqttUnsubscribeRequest unsubscribeRequ
      * @return reply to publish
      * @throws StatusRuntimeException on errors
      */
-    public MqttPublishReply publishMqtt(MqttPublishRequest publishRequest) {
```

Comment:
Could we leave "public" access modifier for all agent control methods? We may need to call them outside package scope

Reply:
Test